src/engine/currencyEngine.js

```diff
@@ -496,7 +503,7 @@ export class CurrencyEngine {
     options?: any = {}
   ): Promise<AbcTransaction> {
     // $FlowFixMe
-    const { privateKeys } = abcSpendInfo
+    const { privateKeys } = { ...abcSpendInfo }
```

Comment:
Since `privateKeys` is an array, this is exactly the same as the code that was changed.
What was the reason for this change?

Reply:
I made a change to try and make flow happy and only half undid it

Reply:
Whatever makes flow happy :)
Technically it's the same code so whatever works with flow is fine.